### PR TITLE
fix(sign-up/login): redirect to login page if sessionId is not valid,…

### LINKED ISSUE #274: Ограничение на длину имени

### PR DESCRIPTION
**Pull request by SeqviriouM**
Redirect to login page if sessionId is not valid. Validation name length during sign up.

Close #274  #249

### CODE REVIEW

File: app/components/SignUp/index.jsx [hunk 1]
```diff
@@ -101,6 +101,16 @@ export default class SignUp extends React.Component {
       });
     }
 
+    if (this.state.name.length > 25) {
+      return this.setState({
```
Comment (dimaip) [inline app/components/SignUp/index.jsx]: Such kind of restrictions on the client without the same limits on the server side just don't make any sense. I could set a breakpoint here and circumvent this protection, filling database with junk.
All kinds of security should be thought server-side first, and only then be reflected in the client interface.

Comment (dimaip) [inline app/components/SignUp/index.jsx]: I.e. we should validate the user data before adding the user to DB here: https://github.com/shri-2015-org/shrimp/blob/fix/sign-up-validation/server/db/db_core.js#L60

**Comment by Nitive** [comment]
Можно ветвиться от сюда, тесты зависли, проблема на стороне CircleCI

File: app/components/SignUp/index.jsx (hunk 1 above)
Comment (SeqviriouM) [inline app/components/SignUp/index.jsx]: yes, we will add server validation soon. But it's another story...

**Comment by Nitive** [comment]
+

**Pull request merged**